vote/Cargo.toml

```diff
@@ -1,58 +0,0 @@
-[package]
-name = "solana-vote"
```

Comment:
nice ya in support of this change.
This is a public crate though do we have to go through the process of deprecating it for a version and then deleting it in the next? or can we just break it